### ISSUE TITLE
question - preg_match nothing to repeat at offset php warning

### ISSUE THREAD

**Issue by chuckreynolds** [foliovision/fv-top-level-cats#8]
This showed up today in php error log. Thoughts? Anything else I could provide to help you figure it out?
```
PHP Warning:  preg_match(): Compilation failed: nothing to repeat at offset 0 in /var/local/xxxxx/xxxxxxx.com/htdocs/wp-content/plugins/fv-top-level-cats/top-level-cats.php on line 98
```

**Comment by foliovision** [comment]
Could you please post:

* what exact permalink structure you are using?
* Is there trailing slash on it or not?
* what exact URL is causing this warning (you can omit your domain name of course)

Thanks,
Martin

**Comment by chuckreynolds** [comment]
sorry just got back from vaca.

permalinks: `/%category%/%postname%/`
trailing slash: yes, always has been
exact url: not sure, that's from server log.

**Comment by foliovision** [comment]
There should be no issues with that permalink structure.

Could you please find the corresponding server access log entry to your error log entry to figure out the broken URL? Or can you run a full site URL structure check and see where it fails? https://foliovision.com/seo-tools/xenu-link-sleuth

**Comment by chuckreynolds** [comment]
I had to hide most of the slugs for anonymity but these are the only 404's that turn up on a robot scrape (using seo frog).  IDK man. I don't see anything that's super weird... I'll keep an eye on it and try and track it down.
![screaming_frog_seo_spider_5_1_ _spider_mode](https://cloud.githubusercontent.com/assets/46185/13159341/1e3d74f4-d646-11e5-85bf-fbdd851daec9.jpg)

**Comment by foliovision** [comment]
Unfortunately the only URLs which you didin't hide appear to be for images.

It appears to me that whenever there is a 404 on your site it triggers this PHP warning in our plugin. Perhaps you are using some plugin (or there is a code in your theme) which makes our plugin do something on the 404 page and that causes the PHP warning.

Could you please try to switch to this version? https://github.com/foliovision/fv-top-level-cats/archive/dev.zip

I tried the following fix in it: https://github.com/foliovision/fv-top-level-cats/commit/2a2daca25b63d1f09b3f72f754676d103c040eb7

**Comment by chuckreynolds** [comment]
dupe comment from patch:
As far as I can tell this works. I'm not seeing the error on our staging environment or my local. The errors were coming from the clone production aws instance but... it seems fine. at least didn't break anything. 
Sorry forgot about this and been busy with other stuff for this client. 
Cheers

**Comment by chuckreynolds** [comment]
also could this have anything to do with Yoast SEO adding back in the `strip category base` setting into /wp-admin/admin.php?page=wpseo_advanced&tab=permalinks ??

**Comment by foliovision** [comment]
We released FV Top Level Categories 1.8.1 with the fix it in.

I'm not sure if Yoast SEO could cause this. We would recommend that you don't enable the option in it if you are using our plugin.
